primary-maternal-care/1B.md

```diff
@@ -1,210 +1,878 @@
 ---
 book: Primary Maternal Care
-title: 1B. Examination of the abdomen in pregnancy
+title: 1. Antenatal care
```

Comment:
@nkoeries What I mean is that you seem to have pasted the content for chapter 1 over Skills chapter 1B here.